src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -71,20 +77,19 @@ public abstract class OptionsBase : TrainerInputBaseWithWeight
             public float StochasticGradientDescentInitilaizationTolerance = 0;
 
             /// <summary>
-            /// Features must occur in at least this many instances to be included
+            /// Determines whether to produce output during training or not.
             /// </summary>
-            /// <remarks>If greater than 1, forces an initialization pass over the data</remarks>
-            //AP removed from now. This requires data transformatio. Perhaps we should handle it as seprate (non-learner) dependant
```

Comment:
shall we keep this piece of info? I bet one = true.

Reply:
diff are not showing well here. The removed text is for CountThreshold which is commented out and I removed it.

---
In reply to: [268038318](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268038318) [](ancestors = 268038318)